guardian/mobile-apps-article-templates
# Replace Outbrain widget with native component

This is the template part of [GLA-1238](https://theguardian.atlassian.net/browse/GLA-1238).

I've pretty much copied the implementation of related content and removed everything related to the existing Outbrain JS widget.

I'm assuming that once this is merged into `master` we *cannot* do any other template changes until https://github.com/guardian/ios-live/tree/gla-1238-outbrain-sdk is also merged into master. Is that correct?

### Testing
Please correct my if I'm wrong in any of the following:

1. Check out this branch locally
2. Build the templates using `npm run build`
3. Check out the [iOS branch](https://github.com/guardian/ios-live/tree/gla-1238-outbrain-sdk) locally
4. Edit `package.json`, assuming that you've checked out the templates in the same parent directory as the iOS project:

```json
"dependencies": {
    "@guardian/mobile-apps-article-templates": "file:../mobile-apps-article-templates"
}
```
5. Run the iOS app *without a subscription*
6. You should see Outbrain recommendations at the bottom of articles.
